examples/js-npm-typescript/README.md

```diff
@@ -0,0 +1,43 @@
+# Web app with Javascript NPM package
+
+This is a basic web application written in TypeScript that uses RudderTyper client library along with JS npm package.
+
+## Prerequisite(If you want to use your tracking plan)
+
+1. You have setup the tracking plan from the dashboard.
+2. Generated PAT from dashboard settings.
+3. Did the authentication step using PAT and email to generate ruddertyper.yml and plan.json file in the root level of the repo.
+4. Move ruddertyper.yml file from root level to examples/js-npm-typescript this folder.
+5. Replace the plan.json in js-npm-typescript/src/analytics with the one you generated.
+6. Delete `index.ts` and `rudder.ts` inside analytics folder.
+7. Run the command `npx rudder-typer`
+8. New `index.ts` and `rudder.ts` file will be generated.
+9. Add `import type { RudderAnalytics, RudderAnalyticsPreloader } from '@rudderstack/analytics-js';` this line at the top of rudder.ts and update rudderanalytics interface type to `rudderanalytics: RudderAnalytics | RudderAnalyticsPreloader | undefined;`
```

Comment:
Let's remove this whole section. Half of them should be part of the repository setup instructions.

Reply:
Addressed